core/geometry/src/test/geometry4d/MomentData.test.ts

```diff
@@ -184,7 +184,7 @@ describe("MomentData", () => {
     expect(ck.getNumErrors()).equals(0);
   });
 
-  it("WireMoments", () => {
+  it.only("WireMoments", () => {

```

Comment:
Remove .only

Reply:
@EarlinLutz

Reply:
Thanks @pmconne  - just pushed that change